deploy/bootstrap.py

```diff
@@ -665,6 +708,10 @@ def _build_jigsaw(options, activate_env, source_path, conda_env_path):
 
 
 def _get_env_vars(machine, compiler, mpilib):
+    """
+    Get the environment variables for the given machine, compiler, and mpi
+    library
+    """
```

Comment:
This seems like another weirdness.  I think we can rename `mpilib` to just `mpi` like elsewhere in the code.  Then:
```suggestion
def _get_env_vars(machine, compiler, mpi):
    """
    Get the environment variables for the given machine, compiler, and MPI
    variant
    """
```

Reply:
Again, I'm fine with cleaning this up here but you could include it in a separate clean-up PR if you prefer.